boards/arm/nrf5340_dk_nrf5340/Kconfig

```diff
@@ -20,6 +20,16 @@ config BOARD_ENABLE_DCDC_HV
 	select SOC_DCDC_NRF53X_HV
 	default y
 
+choice BT_HCI_BUS_TYPE
+	default BT_RPMSG if BT
+endchoice
+
+config HEAP_MEM_POOL_SIZE
+	default 8192 if BT_RPMSG
```

Comment:
@kapi-no this value is not supported  as far as I can see. This seems to be a documentation error, though.
I am just wondering; do you really need that much of heap? Can you live with 4096 and then we update this when we fix the documentation?

Reply:
Changed to 4096, the test samples seem to be working as well. The configuration could be tweaked later on if this is not an optimal number.

Reply:
perfect, thanks